index.js

```diff
@@ -1,126 +1,19 @@
 module.exports = {
-  plugins: ['flowtype', 'import', 'prettier', 'promise', 'react', 'react-hooks'],
-  extends: ['plugin:flowtype/recommended', 'plugin:react/recommended', 'airbnb', 'prettier'],
+  plugins: [
```

Comment:
You don't need that, do you?
```suggestion
  plugins: [
```

Reply:
removed 👍